### PR TITLE
docs: switch to suggesting jsDelivr as a CDN

### LINKED ISSUE #31887: v4.5.3 is not on BootstrapCDN
https://stackpath.bootstrapcdn.com/bootstrap/4.5.3/js/bootstrap.bundle.min.js
https://stackpath.bootstrapcdn.com/bootstrap/4.5.3/css/bootstrap.min.css

Yields error like:

```
<Error>
    <Code>AccessDenied</Code>
    <Message>Access Denied</Message>
    <RequestId>8T8Q3M7J1ZBN1Y8J</RequestId>
    <HostId>
        UoEdHYsY8K4kY7S8P340gWmBZcqUPjlIiE4d5zuHAN+qyA9ZgIOXBsoeSkjGjdQiDygTA3dz9YE=
    </HostId>
</Error>
```

### PR DESCRIPTION
**Pull request by XhmikosR**
Closes #31887 

Preview: <https://deploy-preview-31894--twbs-bootstrap.netlify.app/>

### CODE REVIEW

File: site/content/docs/5.0/getting-started/download.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ If you want to download and examine our [examples]({{< docsref "/examples" >}}),
 
 <a href="{{< param "download.dist_examples" >}}" class="btn btn-bd-primary" onclick="ga('send', 'event', 'Getting started', 'Download', 'Download Examples');">Download Examples</a>
 
-## BootstrapCDN
+## jsDelivr
```
Comment (XhmikosR) [inline site/content/docs/5.0/getting-started/download.md:37]: Maybe we should change this to "CDN"?

File: site/content/docs/5.0/getting-started/introduction.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ toc: true
 
 ## Quick start
 
-Looking to quickly add Bootstrap to your project? Use BootstrapCDN, provided for free by the folks at StackPath. Using a package manager or need to download the source files? [Head to the downloads page]({{< docsref "/getting-started/download" >}}).
+Looking to quickly add Bootstrap to your project? Use jsDelivr, TBF. Using a package manager or need to download the source files? [Head to the downloads page]({{< docsref "/getting-started/download" >}}).
```
Comment (XhmikosR) [inline site/content/docs/5.0/getting-started/introduction.md:15]: Reminder to fill this sentence

File: site/content/docs/5.0/getting-started/download.md (hunk 1 above)
Comment (mdo) [inline site/content/docs/5.0/getting-started/download.md:37]:
```suggestion
## CDN via jsDelivr
```

File: site/content/docs/5.0/getting-started/introduction.md (hunk 2 above)
Comment (mdo) [inline site/content/docs/5.0/getting-started/introduction.md:15]:
```suggestion
Looking to quickly add Bootstrap to your project? Use jsDelivr, a free open source CDN. Using a package manager or need to download the source files? [Head to the downloads page]({{< docsref "/getting-started/download" >}}).
```

**Review by mdo — APPROVED** [review]
Two suggested content edits.

**Review by mdo — APPROVED** [review]

**Pull request merged**